src/metric-engine/src/engine/create.rs

```diff
@@ -539,6 +535,13 @@ impl MetricEngineInner {
     }
 }
 
+/// Creates the region options for metadata region in metric engine.
+pub(crate) fn region_options_for_metadata_region() -> HashMap<String, String> {
+    [(TTL_KEY.to_string(), "10000 years".to_string())]
```

Comment:
I think we must clone the original option and do the stuff.

Reply:
> I think we must clone the original option and do the stuff.

Metadata region for metric engine should stay unaffected from any table level options.

Reply:
I disagree. For example, Maye the user prefers to explicitly set the compaction options. That way, if we add new options in the future, we won't run into unexpected behavior in this region.